private.tf

```diff
@@ -18,7 +18,7 @@ resource "aws_subnet" "private" {
   cidr_block              = cidrsubnet(var.cidr_block, ceil(log(var.max_subnets, 2)), count.index)
   map_public_ip_on_launch = var.map_public_ip_on_launch_enabled
 
-  tags = merge({
+  tags = merge(module.private_label.tags, {
     "Name"      = "${module.private_label.id}${module.this.delimiter}${element(var.subnet_names, count.index)}"
     "Stage"     = module.private_label.stage
```

Comment:
Can you remove the Stage + Namespace entries from the override map? They're unnecessary now that we're using the top level tags as the base map. Please remove them from all of the instances that you've updated.

Reply:
I was actually just doing that.